src/cli/Microsoft.DotNet.UpgradeAssistant.Cli/Program.cs

```diff
@@ -36,9 +37,15 @@ public static Task<int> Main(string[] args)
             root.AddCommand(new ExtensionManagementCommand());
             root.AddCommand(new FeatureFlagCommand());
 
+            var helpBuilder = new HelpBuilder(LocalizationResources.Instance, ConsoleUtils.Width);
+            helpBuilder.CustomizeLayout(_ =>
+                            HelpBuilder.Default
+                                       .GetLayout()
+                                       .Prepend(_ => ShowHeader()));
+
```

Comment:
What is the gain in replacing the custom implementation for help builder? We need to display custom header details which is missing here.

Reply:
Since the implementation here is still making use of the default help builder, this will still show the default of `Upgrade-assistant [options] [command]` which does not address your observation in #1070